diff -aurN abiword-2.4.2.orig/abi/abiword.desktop abiword-2.4.2/abi/abiword.desktop
--- abiword-2.4.2.orig/abi/abiword.desktop	2005-11-20 17:10:15.000000000 +0100
+++ abiword-2.4.2/abi/abiword.desktop	2006-01-11 17:18:30.000000000 +0100
@@ -1,13 +1,12 @@
 [Desktop Entry]
 Exec=abiword
-Icon=abiword_48.png
+Icon=abiword.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=GTK;GNOME;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
-X-Desktop-File-Install-Version=0.9
-MimeType=application/x-abiword;text/x-abiword;text/x-xml-abiword;text/plain;application/vnd.ms-word;application/rtf;application/vnd.plain;application/xhtml+xml;text/html;
+MimeType=application/x-abiword;text/x-abiword;text/x-xml-abiword;
 Name=AbiWord
 GenericName=Word Processor
 Comment=AbiWord Word Processor
@@ -19,7 +18,6 @@
 Comment[nl]=AbiWord tekstverwerker
 Name[nn]=Tekstbehandling
 Comment[nn]=AbiWord Tekstbehandling
-Name[no]=Tekstbehandler
-Comment[no]=AbiWord Tekstbehandler
+Name[pl]=Procesor tekstu AbiWord
 Name[pt]=Processador de Texto
 Comment[pt]=AbiWord Processador de Texto
