--- abiword-2.8.6/abiword.desktop.orig	2010-04-02 21:48:05.000000000 +0200
+++ abiword-2.8.6/abiword.desktop	2011-08-09 12:46:04.744024498 +0200
@@ -1,10 +1,10 @@
 Icon=abiword
 Terminal=false
 Type=Application
-Categories=Office;WordProcessor;GNOME;GTK;X-Red-Hat-Base;
+Categories=GTK;GNOME;Office;WordProcessor;
 StartupNotify=true
 X-Desktop-File-Install-Version=0.9
-MimeType=application/x-abiword;text/x-abiword;text/x-xml-abiword;text/plain;application/msword;application/rtf;application/vnd.plain;application/xhtml+xml;text/html;application/x-crossmark;application/docbook+xml;application/x-t602;application/vnd.oasis.opendocument.text;application/vnd.oasis.opendocument.text-template;application/vnd.oasis.opendocument.text-web;application/vnd.sun.xml.writer;application/vnd.stardivision.writer;text/vnd.wap.wml;application/wordperfect6;application/wordperfect5.1;application/vnd.wordperfect;application/x-abicollab;
+MimeType=application/x-abiword;text/x-abiword;text/x-xml-abiword;application/msword;application/rtf;application/vnd.plain;application/xhtml+xml;text/html;application/x-crossmark;application/docbook+xml;application/x-t602;application/vnd.oasis.opendocument.text;application/vnd.oasis.opendocument.text-template;application/vnd.oasis.opendocument.text-web;application/vnd.sun.xml.writer;application/vnd.stardivision.writer;text/vnd.wap.wml;application/wordperfect6;application/wordperfect5.1;application/vnd.wordperfect;application/x-abicollab;
 Name=AbiWord
 GenericName=Word Processor
 Comment=Compose, edit, and view documents
@@ -28,7 +28,7 @@
 Comment[nl]=AbiWord tekstverwerker
 Comment[nn]=AbiWord Tekstbehandling
 Comment[no]=AbiWord Tekstbehandler
-Comment[pl]=Edytor tekstu AbiWord
+Comment[pl]=Procesor tekstu AbiWord
 Comment[pt]=AbiWord Processador de Texto
 GenericName[ru]=Текстовый редактор
 Comment[ru]=Обработка форматированных текстовых документов
