diff -Nuard abiword-2.0.2.orig/abi/abiword.desktop abiword-2.0.2/abi/abiword.desktop
--- abiword-2.0.2.orig/abi/abiword.desktop	2003-11-05 23:48:22.000000000 +0100
+++ abiword-2.0.2/abi/abiword.desktop	2003-12-15 20:17:02.584553768 +0100
@@ -1,11 +1,14 @@
 [Desktop Entry]
-Name=AbiWord Word Processor
+Name=AbiWord
+Name[pl]=AbiWord
 Name[de]=AbiWord Textverarbeitung
+Comment=Word Processor
+Comment[pl]=Edytor dokumentów tekstowych
 Exec=abiword
 Icon=abiword_48.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=Application;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
 
