diff -urN aa/abiword-2.1.6/abiword-plugins/tools/abipaint/xp/AbiPaint.cpp abiword-2.1.6/abiword-plugins/tools/abipaint/xp/AbiPaint.cpp
--- aa/abiword-2.1.6/abiword-plugins/tools/abipaint/xp/AbiPaint.cpp	2003-10-25 19:17:29.000000000 +0200
+++ abiword-2.1.6/abiword-plugins/tools/abipaint/xp/AbiPaint.cpp	2004-08-28 15:39:56.205852624 +0200
@@ -538,7 +538,7 @@
 				pView->cmdUnselectSelection();
 				pView->setPoint(pos);
 				pView->extSelHorizontal(true, 1); // move point forward one
-				errorCode = pView->cmdInsertGraphic(pFG, szTmpPng.c_str());
+				errorCode = pView->cmdInsertGraphic(pFG);
 				if (errorCode)
 				{
 					pFrame->showMessageBox("Could not put image back into Abiword", XAP_Dialog_MessageBox::b_O,XAP_Dialog_MessageBox::a_OK);
