--- abiword-2.1.7/abi/abiword.desktop.orig	2004-08-26 18:13:35.000000000 +0200
+++ abiword-2.1.7/abi/abiword.desktop	2004-09-09 17:26:09.724425056 +0200
@@ -3,7 +3,7 @@
 Icon=abiword_48.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=GTK:GNOME;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
 X-Desktop-File-Install-Version=0.4
@@ -12,5 +12,5 @@
 Name[de]=AbiWord Textverarbeitung
 Name[nb]=AbiWord Tekstbehandling
 Name[nn]=AbiWord Tekstbehandling
-Name[no]=AbiWord Tekstredigeringsprogram
+Name[pl]=Procesor tekstu AbiWord
 Name[pt]=AbiWord Processador de Texto
