diff -urN abiword-2.1.1.orig/abi/abiword.desktop abiword-2.1.1/abi/abiword.desktop
--- abiword-2.1.1.orig/abi/abiword.desktop	2004-03-26 11:40:33.669215856 +0100
+++ abiword-2.1.1/abi/abiword.desktop	2004-03-26 11:43:08.329703888 +0100
@@ -3,17 +3,18 @@
 Name[de]=Textverarbeitung
 Name[nb]=Tekstbehandling
 Name[nn]=Tekstbehandling
+Name[pl]=Procesor tekstu
 Name[pt]=Processador de Texto
 Comment=AbiWord Word Processor
 Comment[de]=AbiWord Textverarbeitung
 Comment[nb]=Tekstbehandlingsprogrammet AbiWord
 Comment[nn]=Tekstbehandlingsprogrammet AbiWord
+Comment[pl]=Procesor dokumentów tekstowych
 Comment[pt]=Processor de Texto AbiWord
 Exec=abiword
 Icon=abiword_48.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=Application;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
-
