--- abiword-plugins/tools/abigochart/plugin.m4~	2006-10-31 22:25:11.000000000 +0100
+++ abiword-plugins/tools/abigochart/plugin.m4	2006-11-17 13:33:03.318315000 +0100
@@ -23,7 +23,7 @@
 
 LIBGOffice_REQUIRED_VERSION=0.2.1
 
-PKG_CHECK_MODULES(GOFFICE, [libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION],[
+PKG_CHECK_MODULES(GOFFICE, [libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION],[
         ABI_PLUGIN_REPORT([abigochart: okay])
 ],[        ABI_PLUGIN_REPORT([abigochart: error libgoffice >= $LIBGOffice_REQUIRED_VERSION required])
         abi_plugin_disable=yes
--- abiword-plugins/tools/abigochart/unix/AbiGOChart.cpp~	2006-10-31 22:25:12.000000000 +0100
+++ abiword-plugins/tools/abigochart/unix/AbiGOChart.cpp	2006-11-17 13:35:44.258315000 +0100
@@ -72,7 +72,7 @@
 #include <goffice/graph/gog-data-allocator.h>
 #include <goffice/graph/gog-series.h>
 #include <goffice/graph/gog-guru.h>
-#include <goffice/graph/gog-renderer-pixbuf.h>
+#include <goffice/graph/gog-renderer-cairo.h>
 #include <goffice/graph/gog-data-set.h>
 #include <goffice/graph/gog-object-xml.h>
 #include <goffice/data/go-data-simple.h>
@@ -1119,7 +1119,7 @@
 {
 	m_Graph = NULL;
 	m_Image = NULL;
-	m_Renderer = GOG_RENDERER (g_object_new (GOG_RENDERER_PIXBUF_TYPE, NULL));
+	m_Renderer = GOG_RENDERER (g_object_new (GOG_RENDERER_CAIRO_TYPE, NULL));
 	pix_width = pix_height = 0;
 	width = height = 0;
 	m_Guru = NULL;
@@ -1189,10 +1189,10 @@
 			GdkPixbuf *pixbuf;
 			pix_width = _width;
 			pix_height = _height;
-			gog_renderer_pixbuf_update (GOG_RENDERER_PIXBUF (m_Renderer), _width, _height, 1.0);	
+			gog_renderer_cairo_update (GOG_RENDERER_CAIRO (m_Renderer), _width, _height, 1.0);	
 			if (m_Image)
 				delete m_Image;
-			pixbuf = gog_renderer_pixbuf_get (GOG_RENDERER_PIXBUF (m_Renderer));
+			pixbuf = gog_renderer_cairo_get_pixbuf (GOG_RENDERER_CAIRO (m_Renderer));
 			m_Image = new GR_UnixImage(NULL, pixbuf);
 			g_object_ref(pixbuf);
 			m_Image->scaleImageTo(m_pGOMan->getGraphics(),rec);
--- abiword-plugins/configure.orig	2006-11-17 15:05:35.228315000 +0100
+++ abiword-plugins/configure	2006-11-17 15:14:15.268315000 +0100
@@ -22834,23 +22834,23 @@
   else
      PKG_CONFIG_MIN_VERSION=0.9.0
      if $PKG_CONFIG --atleast-pkgconfig-version $PKG_CONFIG_MIN_VERSION; then
-        echo "$as_me:$LINENO: checking for libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION" >&5
-echo $ECHO_N "checking for libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION... $ECHO_C" >&6
+        echo "$as_me:$LINENO: checking for libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION" >&5
+echo $ECHO_N "checking for libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION... $ECHO_C" >&6
 
-        if $PKG_CONFIG --exists "libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION" ; then
+        if $PKG_CONFIG --exists "libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION" ; then
             echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
             succeeded=yes
 
             echo "$as_me:$LINENO: checking GOFFICE_CFLAGS" >&5
 echo $ECHO_N "checking GOFFICE_CFLAGS... $ECHO_C" >&6
-            GOFFICE_CFLAGS=`$PKG_CONFIG --cflags "libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION"`
+            GOFFICE_CFLAGS=`$PKG_CONFIG --cflags "libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION"`
             echo "$as_me:$LINENO: result: $GOFFICE_CFLAGS" >&5
 echo "${ECHO_T}$GOFFICE_CFLAGS" >&6
 
             echo "$as_me:$LINENO: checking GOFFICE_LIBS" >&5
 echo $ECHO_N "checking GOFFICE_LIBS... $ECHO_C" >&6
-            GOFFICE_LIBS=`$PKG_CONFIG --libs "libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION"`
+            GOFFICE_LIBS=`$PKG_CONFIG --libs "libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION"`
             echo "$as_me:$LINENO: result: $GOFFICE_LIBS" >&5
 echo "${ECHO_T}$GOFFICE_LIBS" >&6
         else
@@ -22858,7 +22858,7 @@
             GOFFICE_LIBS=""
             ## If we have a custom action on failure, don't print errors, but
             ## do set a variable so people can do so.
-            GOFFICE_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION"`
+            GOFFICE_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "libgoffice-0.5 >= $LIBGOffice_REQUIRED_VERSION"`
 
         fi
 
