--- abiword-2.0.14/abiword-plugins/tools/gypsython/plugin.m4.orig	2004-12-23 14:01:57.074696040 +0100
+++ abiword-2.0.14/abiword-plugins/tools/gypsython/plugin.m4	2004-12-23 14:02:20.946626000 +0100
@@ -28,27 +28,27 @@
 # Checks for libraries.
 # Checks for header files.
 
-AC_ARG_WITH(python,[  --with-python=DIR        Python 2.3 library in DIR],[
+AC_ARG_WITH(python,[  --with-python=DIR        Python 2.4 library in DIR],[
 	if test "x$withval" = "xno"; then
 		ABI_PLUGIN_REPORT([gypsython: Python support disabled])
 		abi_plugin_disable=yes
 	elif test "x$withval" != "xyes"; then
-		ABI_GYPSY_CPPFLAGS="-I$withval/include/python2.3"
+		ABI_GYPSY_CPPFLAGS="-I$withval/include/python2.4 -I$withval/include/python2.3"
 		ABI_GYPSY_LDFLAGS="-L$withval/lib"
-		CPPFLAGS="$CPPFLAGS -I$withval/include/python2.3"
+		CPPFLAGS="$CPPFLAGS -I$withval/include/python2.4 -I$withval/include/python2.3"
 		LDFLAGS="$LDFLAGS -L$withval/lib"
 	else
-		ABI_GYPSY_CPPFLAGS="-I/usr/include/python2.3"
-		CPPFLAGS="$CPPFLAGS -I/usr/include/python2.3"
+		ABI_GYPSY_CPPFLAGS="-I/usr/include/python2.4 -I/usr/include/python2.3"
+		CPPFLAGS="$CPPFLAGS -I/usr/include/python2.4 -I/usr/include/python2.3"
 	fi
-],[	ABI_GYPSY_CPPFLAGS="-I/usr/include/python2.3"
-	CPPFLAGS="$CPPFLAGS -I/usr/include/python2.3"
+],[	ABI_GYPSY_CPPFLAGS="-I/usr/include/python2.4 -I/usr/include/python2.3"
+	CPPFLAGS="$CPPFLAGS -I/usr/include/python2.4 -I/usr/include/python2.3"
 ])
 
 if test $abi_plugin_disable != yes; then
 	AC_CHECK_HEADER(Python.h,[
-		AC_CHECK_LIB(python2.3,Py_Initialize,[
-			ABI_GYPSY_LDFLAGS="$ABI_GYPSY_LDFLAGS -lpython2.3"
+		AC_CHECK_LIB(python,Py_Initialize,[
+			ABI_GYPSY_LDFLAGS="$ABI_GYPSY_LDFLAGS -lpython"
 			ABI_PLUGIN_REPORT([gypsython: okay])
 		],[	ABI_PLUGIN_REPORT([gypsython: error - Python library not found])
 			abi_plugin_disable=yes
--- abiword-2.0.14/abiword-plugins/tools/gypsython/xp/Gypsython.h.orig	2004-12-23 14:10:50.111645632 +0100
+++ abiword-2.0.14/abiword-plugins/tools/gypsython/xp/Gypsython.h	2004-12-23 14:11:02.641740768 +0100
@@ -37,7 +37,7 @@
 #include "xap_Dlg_Language.h"
 #include "xap_Frame.h"
 
-#include <python2.3/Python.h>
+#include <Python.h>
  
  
  
