--- abiword-2.6.8/abiword-plugins-2.6.8/tools/abicommand/xp/AbiCommand.cpp~	2007-10-11 22:05:55.000000000 +0200
+++ abiword-2.6.8/abiword-plugins-2.6.8/tools/abicommand/xp/AbiCommand.cpp	2009-07-14 17:02:04.691251879 +0200
@@ -551,7 +551,7 @@
 				UT_UTF8String *pCom1 =
 					const_cast < UT_UTF8String * >(static_cast < const UT_UTF8String * >(pToks->getNthItem (1)));
 				printf(" Filename %s \n",pCom1->utf8_str());
-				char *suffix = rindex (pCom1->utf8_str (), '.');
+				char *suffix = (char *)rindex (pCom1->utf8_str (), '.');
 
 				if (suffix != NULL)
 				{
--- abiword-2.6.8/src/wp/ap/unix/GNUmakefile.am~	2007-11-04 15:46:12.000000000 +0100
+++ abiword-2.6.8/src/wp/ap/unix/GNUmakefile.am	2009-07-14 17:35:39.114130442 +0200
@@ -58,7 +58,6 @@
 	ap_UnixDialog_MailMerge.glade	\
 	ap_UnixDialog_MetaData.glade	\
 	ap_UnixDialog_Options.glade	\
-	ap_UnixDialog_HdrFtr.glade \
 	ap_UnixDialog_Options_ColorSel.glade	\
 	ap_UnixDialog_PageNumbers.glade		\
 	ap_UnixDialog_PageSetup.glade		\
--- abiword-2.6.8/src/wp/ap/unix/GNUmakefile.am~	2009-07-14 17:35:39.114130442 +0200
+++ abiword-2.6.8/src/wp/ap/unix/GNUmakefile.am	2009-07-14 17:37:17.458076101 +0200
@@ -57,7 +57,6 @@
 	ap_UnixDialog_Insert_DateTime.glade \
 	ap_UnixDialog_MailMerge.glade	\
 	ap_UnixDialog_MetaData.glade	\
-	ap_UnixDialog_Options.glade	\
 	ap_UnixDialog_Options_ColorSel.glade	\
 	ap_UnixDialog_PageNumbers.glade		\
 	ap_UnixDialog_PageSetup.glade		\
