--- abiword-2.1.5/abi/abiword.desktop.orig	2004-07-31 19:57:52.000000000 +0200
+++ abiword-2.1.5/abi/abiword.desktop	2004-08-12 18:56:11.980696248 +0200
@@ -3,7 +3,7 @@
 Icon=abiword_48.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=Application;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
 X-Desktop-File-Install-Version=0.4
@@ -18,5 +18,7 @@
 Comment[nn]=Tekstbehandlingsprogrammet AbiWord
 Name[no]=Tekstredigeringsprogram
 Comment[no]=AbiWord Tekstredigeringsprogram
+Name[pl]=Procesor tekstu
+Comment[pl]=Procesor dokumentów tekstowych
 Name[pt]=Processador de Texto
 Comment[pt]=Processor de Texto AbiWord
