diff -aurN abiword-2.1.90.orig/abi/abiword.desktop abiword-2.1.90/abi/abiword.desktop
--- abiword-2.1.90.orig/abi/abiword.desktop	2004-09-30 16:34:58.000000000 +0200
+++ abiword-2.1.90/abi/abiword.desktop	2004-10-16 00:01:20.331450440 +0200
@@ -3,14 +3,13 @@
 Icon=abiword_48.png
 Terminal=false
 Type=Application
-Categories=Application;Office;WordProcessor;X-Red-Hat-Base;
+Categories=GTK:GNOME;Office;WordProcessor;
 StartupNotify=true
 Encoding=UTF-8
-X-Desktop-File-Install-Version=0.4
-MimeType=application/x-abiword; text/x-abiword; text/x-xml-abiword; text/plain;
+MimeType=application/x-abiword;text/x-abiword;text/x-xml-abiword;text/plain;
 Name=AbiWord Word Processor
 Name[de]=AbiWord Textverarbeitung
 Name[nb]=AbiWord Tekstbehandling
 Name[nn]=AbiWord Tekstbehandling
-Name[no]=AbiWord Tekstbehandler
+Name[pl]=Procesor tekstu AbiWord
 Name[pt]=AbiWord Processador de Texto
