--- abiword-2.4.4/abiword-plugins/wp/impexp/xhtml/xp/ie_imp_MHT.cpp.orig	2006-06-28 18:43:26.000000000 +0000
+++ abiword-2.4.4/abiword-plugins/wp/impexp/xhtml/xp/ie_imp_MHT.cpp	2006-06-28 18:44:12.000000000 +0000
@@ -159,7 +159,7 @@
 
 	bool bValid = false;
 
-	for (struct header_t * h = eps_next_header (m_eps); h; h = eps_next_header (m_eps))
+	for (header_t * h = eps_next_header (m_eps); h; h = eps_next_header (m_eps))
 		{
 			const char * name = reinterpret_cast<char *>(h->name);
 			const char * data = reinterpret_cast<char *>(h->data);
@@ -359,7 +359,7 @@
 	UT_Multipart * part = new UT_Multipart;
 	if (part == 0) return 0;
 
-	for (struct header_t * h = mime_next_header (m_eps); h; h = mime_next_header (m_eps))
+	for (header_t * h = mime_next_header (m_eps); h; h = mime_next_header (m_eps))
 		{
 			const char * name = reinterpret_cast<char *>(h->name);
 			const char * data = reinterpret_cast<char *>(h->data);
--- abiword-2.4.4/abiword-plugins/wp/impexp/xhtml/xp/ie_imp_MHT.h.orig	2006-06-28 18:34:08.000000000 +0000
+++ abiword-2.4.4/abiword-plugins/wp/impexp/xhtml/xp/ie_imp_MHT.h	2006-06-28 18:34:47.000000000 +0000
@@ -138,7 +138,7 @@
 	UT_Multipart *	m_document;
 	UT_Vector *		m_parts;
 
-	struct eps_t *	m_eps;
+	eps_t *	m_eps;
 };
 
 class MultiReader : public UT_XML::Reader

