--- abiword-2.2.8.orig/abi/src/wp/impexp/xp/ie_mailmerge.h	2005-01-23 15:37:50.000000000 +0100
+++ abiword-2.2.8/abi/src/wp/impexp/xp/ie_mailmerge.h	2005-06-06 17:19:34.932480728 +0200
@@ -28,6 +28,8 @@
 typedef UT_sint32 IEMergeType;
 #define IEMT_Unknown ((IEMergeType)-1)
 
+class ABI_EXPORT IE_MailMerge;
+
 class ABI_EXPORT IE_MergeSniffer : public UT_AbiObject
 {
   friend class IE_MailMerge;
