diff -aurN abiword-plugins.orig/tools/abigochart/plugin.m4 abiword-plugins/tools/abigochart/plugin.m4
--- abiword-plugins.orig/tools/abigochart/plugin.m4	2005-12-27 12:09:54.000000000 +0100
+++ abiword-plugins/tools/abigochart/plugin.m4	2006-06-11 19:18:51.900687200 +0200
@@ -23,7 +23,7 @@
 
 LIBGOffice_REQUIRED_VERSION=0.1.0
 
-PKG_CHECK_MODULES(GOFFICE, [libgoffice-1 >= $LIBGOffice_REQUIRED_VERSION],[
+PKG_CHECK_MODULES(GOFFICE, [libgoffice-0.3 >= $LIBGOffice_REQUIRED_VERSION],[
         ABI_PLUGIN_REPORT([abigochart: okay])
 ],[        ABI_PLUGIN_REPORT([abigochart: error libgoffice >= $LIBGOffice_REQUIRED_VERSION required])
         abi_plugin_disable=yes
diff -aurN abiword-plugins.orig/tools/abigochart/unix/AbiGOChart.cpp abiword-plugins/tools/abigochart/unix/AbiGOChart.cpp
--- abiword-plugins.orig/tools/abigochart/unix/AbiGOChart.cpp	2006-03-26 15:28:54.000000000 +0200
+++ abiword-plugins/tools/abigochart/unix/AbiGOChart.cpp	2006-06-11 19:56:15.363628992 +0200
@@ -1256,7 +1256,7 @@
 	/* the resolution of symbols in libgoffice must be forced because when
 	creating the guru window, libglade needs foo_canvas_new. There might
 	be a better solution..., I could not find one */
-	g_module_close (g_module_open ("libgoffice-1", (GModuleFlags)0));
+	g_module_close (g_module_open ("libgoffice-0.3", (GModuleFlags)0));
    return 1;
 }
 
