diff -urN aa/abiword-2.1.2/abi/ac-helpers/abi-gucharmap.m4 abiword-2.1.2/abi/ac-helpers/abi-gucharmap.m4
--- aa/abiword-2.1.2/abi/ac-helpers/abi-gucharmap.m4	2004-02-20 00:27:04.000000000 +0100
+++ abiword-2.1.2/abi/ac-helpers/abi-gucharmap.m4	2004-04-30 23:49:42.947548184 +0200
@@ -33,7 +33,7 @@
 ])
 
 if test "x$test_cmap" = "xtrue" ; then
-	PKG_CHECK_MODULES(GUCHARMAP,[gucharmap >= 0.7, gucharmap < 1.3.0], [have_cmap=true], [have_cmap=false])
+	PKG_CHECK_MODULES(GUCHARMAP,[gucharmap >= 1.4.0], [have_cmap=true], [have_cmap=false])
 fi
 
 if test "x$have_cmap" = "xtrue" ; then
diff -urN aa/abiword-2.1.2/abi/src/af/xap/unix/xap_UnixDlg_Insert_Symbol.cpp abiword-2.1.2/abi/src/af/xap/unix/xap_UnixDlg_Insert_Symbol.cpp
--- aa/abiword-2.1.2/abi/src/af/xap/unix/xap_UnixDlg_Insert_Symbol.cpp	2004-04-12 20:48:05.000000000 +0200
+++ abiword-2.1.2/abi/src/af/xap/unix/xap_UnixDlg_Insert_Symbol.cpp	2004-04-30 23:48:56.669583504 +0200
@@ -522,7 +522,7 @@
 	m_areaCurrentSym = _previewNew (60, 45);
 	gtk_box_pack_start(GTK_BOX(hbox), m_areaCurrentSym, TRUE, FALSE, 0);
 #else
-	m_SymbolMap = gucharmap_charmap_new ();
+	m_SymbolMap = gucharmap_charmap_new (GUCHARMAP_CHAPTERS(gucharmap_block_chapters_new()));
 	gtk_widget_show (m_SymbolMap);
 	gtk_box_pack_start (GTK_BOX (tmp), m_SymbolMap, TRUE, TRUE, 0);
 	gtk_container_set_border_width (GTK_CONTAINER (m_SymbolMap), 5);
