diff -urN abiword-plugins/wp/impexp/graphics/magick/xp/AbiMagick.cpp abiword-plugins-old/wp/impexp/graphics/magick/xp/AbiMagick.cpp
--- abiword-plugins/wp/impexp/graphics/magick/xp/AbiMagick.cpp	Mon Apr 29 19:45:35 2002
+++ abiword-plugins-old/wp/impexp/graphics/magick/xp/AbiMagick.cpp	Wed Mar 27 19:31:42 2002
@@ -44,16 +44,11 @@
 static Magick::Image ImageFromPNGByteBuf ( const UT_ByteBuf * pBB )
 {
   Magick::Blob inBlob ( pBB->getPointer (0), pBB->getLength () );
-
-  // not Magick::Image ( inBlob ) which doesn't work on win32 for some reason
-  Magick::Image img ;
-  img.read ( inBlob ) ;
-
-  return img ;
+  return Magick::Image ( inBlob );
 }
 
-static UT_Error PNGByteBufFromImage ( Magick::Image & inImage,
-				      UT_ByteBuf ** outByteBuf )
+static void PNGByteBufFromImage ( Magick::Image & inImage,
+				  UT_ByteBuf ** outByteBuf )
 {
   Magick::Blob outBlob;
   inImage.write ( &outBlob, "png" );
@@ -63,8 +58,6 @@
   buf->append ( (const UT_Byte*)outBlob.data (), outBlob.length () );
 
   *outByteBuf = buf ;
-
-  return UT_OK ;
 }
 
 // -----------------------------------------------------------------------
@@ -128,7 +121,9 @@
   {
     try
       {	
-	return PNGByteBufFromImage ( ImageFromPNGByteBuf ( pBB ), ppBB ) ;
+	Magick::Image img ( ImageFromPNGByteBuf ( pBB ) ) ;
+	PNGByteBufFromImage ( img, ppBB ) ;
+	return UT_OK;
       }
     catch (...)
       {
