diff -aurN abiword-2.4.2.orig/abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp abiword-2.4.2/abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp
--- abiword-2.4.2.orig/abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp	2005-06-29 04:22:13.000000000 +0200
+++ abiword-2.4.2/abiword-plugins/wp/impexp/pdf/xp/ie_imp_PDF.cpp	2006-03-23 21:21:23.777224992 +0100
@@ -75,7 +75,7 @@
       UT_tmpnam(temporary_file);
 
       TextOutputDev *textOut = new TextOutputDev(temporary_file, gFalse, gFalse, gFalse);
-      doc->displayPages(textOut, 1, doc->getNumPages(), 72, 72, 0, gTrue, gFalse);
+      doc->displayPages(textOut, 1, doc->getNumPages(), 72, 72, 0, gTrue, gFalse, gFalse);
       delete textOut;
 
       IE_Imp_Text txt_imp(getDoc(), "UTF-8");      
