--- abiword-3.0.2/configure.ac~	2016-10-21 02:38:12.000000000 +0200
+++ abiword-3.0.2/configure.ac	2019-10-19 23:08:25.758721467 +0200
@@ -477,7 +477,7 @@ AM_CONDITIONAL([WITH_REDLAND], test "$ab
 AM_CONDITIONAL([HAVE_REDLAND], test "$abi_cv_redland" = "yes")
 
 
-evolution_data_server_req='libebook-1.2 >= 3.6 libecal-1.2'
+evolution_data_server_req='libebook-1.2 >= 3.6 libecal-2.0'
 AC_ARG_WITH([evolution_data_server], 
 	[AS_HELP_STRING([--with-evolution-data-server], [Use Evolution Data Server to get at contact and calendar information])], 
 [
